src/net/java/sip/communicator/plugin/jabberaccregwizz/IceConfigPanel.java

```diff
@@ -202,14 +205,25 @@ public void actionPerformed(ActionEvent e)
 
                 if (stunServer != null)
                 {
+                    String userName;
+                    String password;
+
+                    try {
+                        userName = StringUtils.toString(
```

Comment:
This is a deprecated method. Use new String(..., StandardCharets.UTF_8) or if it throws use toEncodedString